react-components/src/query/network/types.ts

```diff
@@ -1,5 +1,10 @@
-import { type ViewDefinition } from '@cognite/sdk';
+import { type AnnotationsInstanceRef, type IdEither, type ViewDefinition } from '@cognite/sdk';
 
 export type DMSView = {
   rawView: ViewDefinition;
 };
+
+export type ClassicAnnotationIdAndAssetReference = {
+  assetReference: IdEither | AnnotationsInstanceRef;
```

Comment:
Could we just use the `InstanceReference` type here? Also, I think this type is very specific, so perhaps it can be put in one of the files where it is being used?